docs/core-concepts/app-deployment.md

```diff
@@ -1,31 +1,283 @@
----
-title: .NET Core Application Deployment concepts
-description: .NET Core Application Deployment concepts
-keywords: .NET, .NET Core
-author: dotnet-bot
-manager: wpickett
-ms.date: 06/20/2016
-ms.topic: article
-ms.prod: .net-core
-ms.technology: .net-core-technologies
-ms.devlang: dotnet
-ms.assetid: da7a31a0-8072-4f23-82aa-8a19184cb701
----
-
-# 🔧 .NET Core Application Deployment concepts
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/471) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
-	
+----
+-title: .NET Core Application Deployment concepts
+-description: .NET Core Application Deployment concepts
+-keywords: .NET, .NET Core
+-author: rpetusha
+-manager: wpickett
+-ms.date: 06/20/2016
+-ms.topic: article
+-ms.prod: .net-core
+-ms.technology: .net-core-technologies
+-ms.devlang: dotnet
+-ms.assetid: da7a31a0-8072-4f23-82aa-8a19184cb701
+----
+# .NET Core Application Deployment #
+
+You can deploy your .NET Core app in either of two ways:
+
+- As a portable app. A portable app relies on a shared system-wide version of the .NET Core framework that is present on a system. Portable applications are .dll filest hat can be launched by using the [dotnet utility](https://docs.microsoft.com/en-us/dotnet/core/core-concepts/core-sdk/cli/dotnet "dotnet utility") from the command line. For example, `dotnet app.dll` runs a portable application named `app`.
+
+- As a self-contained application. All components, including .NET Core itself, are included with the application and are isolated from other .NET Core applications. Self-contained applications include an executable (such as `app.exe` for a self-contained application named `app`), which is  a renamed version of the platform-specific .NET Core host, and a .dll file (such as `app.dll`), which is the actual application.
```

Comment:
The executable is going to have the .exe extension only on Windows. Is that worth pointing out?